en/development/configuration.rst

```diff
@@ -51,7 +51,7 @@ datasource
 persistent
     Whether or not to use a persistent connection to the database. When using
     SQLServer you should not enable persistent connections as it causes
-    difficult to diagnose crashes.
+    difficulty to diagnose crashes.
```

Comment:
Do you mean I should remove this change? I thought "cause" is transitive verb and should be followed by a noun instead of an adjective.